composer.json

```diff
@@ -0,0 +1,17 @@
+{
+	"name": "damejidlo/datetime-factory",
+	"description": "",
+	"type": "library",
+	"require": {
+		"php": ">= 5.6.0",
+		"nette/utils": "2.3.*"
+	},
+	"require-dev": {
+		"nette/tester": "@dev"
```

Comment:
Hele jestli ti to udělá radost, já bych klidně všude psal `@dev` a na jel na roling updates a (fuck stability) :D